specification/sql/resource-manager/readme.md

```diff
@@ -140,6 +138,8 @@ input-file:
 - Microsoft.Sql/preview/2019-06-01-preview/managedDatabases.json
 - Microsoft.Sql/preview/2019-06-01-preview/ServerAzureADAdministrators.json
 - Microsoft.Sql/preview/2019-06-01-preview/managedInstanceOperations.json
+- Microsoft.Sql/preview/2019-06-01-preview/syncGroups.json
```

Comment:
Please make sure it's in the following sections:
- package-composite-v3
- package-composite-v2
- package-composite-v1
- package-pure-2019-06-preview